stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -97,11 +101,13 @@ public func _stdlib_select(
         let readAddr = readfds.baseAddress
         let writeAddr = writefds.baseAddress
         let errorAddr = errorfds.baseAddress
+        let bindAsFdSet = { (p: UnsafeMutablePointer<UInt>?) in
```

Comment:
This seems to deserve a real function.  You can define it right here if you want, y’know.